### ISSUE TITLE
Empty Config File

### ISSUE THREAD

**Issue by alirezatbp** [Undeadkillz/node-steam-bot-manager#43]
Hi, I just installed the package using npm and wanted to test one of the example files... when i run it, it creates a config folder containing config.json and a data folder. but both of them are empty and i get this error from node:
 2019-01-18T12:40:09.962Z - error: Failed to validate -  Error=Failed to parse due toSyntaxError: Unexpected end of JSON input
2019-01-18T12:40:09.967Z - error: Failed to load config

**Comment by SamerAlsayegh** [comment]
Yeah this is an issue with the coding, I will be reviewing the code later. In the meantime, you may just copy this pastebin in the config and it should work:

https://pastebin.com/46zLY3AW

Please let me know if you have any other problems.

**Comment by alirezatbp** [comment]
That works for the config file. what about the statistic file? that one's empty also
